stubs/_blueman.pyi

```diff
@@ -0,0 +1,58 @@
+from typing import Any, List, Dict, Union, Callable
+from mypy_extensions import TypedDict
+
+ERR: Dict[int, str]
+RFCOMM_HANGUP_NOW: int
+RFCOMM_RELEASE_ONHUP: int
+RFCOMM_REUSE_DLC: int
+RFCOMM_STATES: List[str]
+RFCOMM_TTY_ATTACHED: int
+
+class RfcommDev(TypedDict):
+    id: int
+    channel: int
+    flags: int
+    state: str
+    src: str
+    dst: str
+
+class HciInfo(TypedDict):
+    acl_mtu: int
+    acl_plts: int
+    bdaddr: str
+    dev_id: int
+    features: List[int]
+    flags: int
+    link_mode: int
+    link_policy: int
+    name: str
+    pkt_type: int
+    sco_mtu: int
+    stat: Dict[str, int]
+    type: int
+
+class BridgeException(Exception):
+    errno: int
+    def __init__(self, errno: int) -> None: ...
+
+class ConnInfoReadError(Exception): ...
+
+class RFCOMMError(Exception): ...
+
+class conn_info:
+    failed: bool
+    def __init__(self, addr: str, hci_name: str) -> None: ...
+    def deinit(self) -> None: ...
+    def get_lq(self) -> int: ...
+    def get_rssi(self) -> int: ...
+    def get_tpl(self) -> int: ...
+    def init(self) -> None: ...
+
+def create_bridge(name: str) -> None: ...
+def create_rfcomm_device(local_address: str, remote_address: str, channel: int) -> int: ...
+def destroy_bridge(name: str) -> None: ...
+def device_info(hci_name: str) -> HciInfo: ...
+def page_timeout(hci_name: str) -> int: ...
+def get_rfcomm_channel(uuid: str, bdaddr: str) -> int: ...
```

Comment:
Might return `None`, so it's `Optional[int]`.

Reply:
Not sure, the c function will always return ``port_num`` which defaults to zero. And if for some reason we try to pass ``None`` as bdaddr we are sure to raise an exception encoding it to UTF8. Am I missing something?

Reply:
I think you're right, the `if bdaddr is not None:` in that function just does not make sense as it's always the case...

Reply:
Yeah, I think I am responsible for the not making sense part when I added python3 support (bytes vs codepoints). I opened #1129 as a reminder for myself to fix this.